test/test_quote.py

```diff
@@ -22,10 +22,10 @@ def test_table_name_with_reserved_words(engine_testaccount, db_parameters):
         inspector = inspect(engine_testaccount)
         columns_in_insert = inspector.get_columns(test_table_name)
         assert len(columns_in_insert) == 3
-        assert columns_in_insert[0]['autoincrement'], 'autoincrement'
-        assert columns_in_insert[0]['default'] is None, 'default'
-        assert columns_in_insert[0]['name'] == 'id', 'name'
-        assert columns_in_insert[0]['primary_key'], 'primary key'
+        assert columns_in_insert[0]['autoincrement'] is False
```

Comment:
for more context on the test change, `autoincrement` and `default` are mutually exclusive in snowflake according to [doc](https://docs.snowflake.com/en/sql-reference/sql/create-table.html#optional-parameters).

by fixing the `sequence` default value issue, we need to update this test accordingly.